src/components/accordion/AccordionMenuSection.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FC, PropsWithChildren } from 'react';
+
+import {
+  AccordionItemProps as ChakraAccordionItemProps,
+  Hide,
+  Show,
+} from '@chakra-ui/react';
+
+import AccordionPanel from './AccordionPanel';
+import AccordionItem from './AccordionItem';
+import AccordionButton from './AccordionButton';
+import Box from '../box';
+
+interface AccordionMenuItemProps extends ChakraAccordionItemProps {
+  title: string;
+}
+
+const AccordionMenuSection: FC<PropsWithChildren<AccordionMenuItemProps>> = (
+  props
+) => {
+  const { children, title, ...itemProps } = props;
+
+  return (
+    <>
+      <Show below="sm">
```

Comment:
I find this a bit weird to be in a component that is exported. The developer that uses this component doesn't know it's mobile-only without checking the implementation detail. I would expect this logic to be part of the footer component and not an exported one. (we can still have it in a separate file, but I wouldn't export it)